### ISSUE TITLE
Move bundled templates into sub-folder

### ISSUE THREAD

**Issue by humdingerb** [Genio-The-Haiku-IDE/Genio#253]
While new users will appreciate having templates for a quick start, over time they will probably adapt them and have their own templates. In that case, having all those bundled templates in a "Bundled templates" sub-folder under "New" would 'hide' them while still being quickly accessible.

I'd also suggest to have a "Show bundled templates..." menu item at the bottom of that sub-folder that shows the bundled templates folder in Tracker. That way you can quickly copy a template to the user templates as basis for further customization.

**Comment by nexus6-haiku** [comment]
My personal stance on this is that if you need to customise a "bundled" template you would open it in Genio, customise it, open the user template folder and the customised folder and copy it.
Further on this, I think there was a plan to add an option to turn a project into a template without ever opening the user folder. Would it be more convenient?

**Comment by humdingerb** [comment]
> My personal stance on this is that if you need to customise a "bundled" template you would open it in Genio, customise it, open the user template folder and the customised folder and copy it.

Yeah, that's how I expect people will do that, if they don't just make a template from their own files without Genio's template as basis.
This ticket is about how Genio's templates will always be in the top level of the "New" menu, even if the user always uses their own customized templates. See this "New" menu example:

![templates](https://github.com/Genio-The-Haiku-IDE/Genio/assets/6107158/4627090a-6456-4482-8fa9-4b8362bf5c20)

All between the "New folder" and "App icon" are Genio templates that the users have to parse and mouse-down until they get to their custom templates. BTW, those user templates should also show the "folders" (= project templates containing files) first, before the template files.

If instead all Genio templates would be in a "Bundled templates" or "Genio templates" subfolder, they are still available, but aren't in the way when the users have their own.

> Further on this, I think there was a plan to add an option to turn a project into a template without ever opening the user folder. Would it be more convenient?

That doesn't address this ticket's issue, but could be useful. It'd simply copy the current file/folder to `"/boot/home/config/settings/Genio/templates/"`.
OTOH, you don't often create new templates. When you're new to Genio you create a few, but then fewer and fewer. The menu item will become less useful and may not be worth taking up the space.
